frontend/src/hooks/pims-api/useFullyAttributedParcelMapLayer.ts

```diff
@@ -20,5 +21,38 @@ export const useFullyAttributedParcelMapLayer = (url: string, name: string) => {
     [getAllFeatures],
   );
 
-  return { findByLegalDescription, findByLegalDescriptionLoading: getAllFeaturesLoading };
+  const findByPid = useCallback(
+    async (pid: string) => {
+      // Removes dashes to match expectations of the map layer.
+      const formattedPid = pid.replace(/-/g, '');
+      const data = await getAllFeatures({ PID: formattedPid }, { forceSimplePid: true });
+
+      return data;
+    },
+    [getAllFeatures],
+  );
+
+  const findByPin = useCallback(
```

Comment:
do you think this should be findAllByPin? the old version of this function had a boolean flag to choose between the two different versions. This function always does an allby search.

Reply:
Good question. Lets have a chat about since this changes how we are retrieving the properties